gojavahash/gojavahash_test.go

```diff
@@ -0,0 +1,93 @@
+package gojavahash
+
+import (
+	"errors"
+	. "github.com/smartystreets/goconvey/convey"
+	"math/rand"
+	"net"
+	"testing"
+	"time"
+)
+
+func TestGoJavaHash(t *testing.T) {
+	Convey("With a new JavaServerSelector", t, func() {
+		js := &JavaServerSelector{}
+		a, err := js.PickServer("test")
+		So(a, ShouldBeNil)
+		So(err, ShouldNotBeNil)
+		Convey("Add Single Server should work", func() {
+			err := js.AddServer("processdata-mc-lab2--aaaa.int.signalfuse.com:11211")
```

Comment:
shouldn't the hostnames be generic names (maybe based on testcase) since this is a public repo?